binaries/daemon/src/spawn.rs

```diff
@@ -149,21 +162,82 @@ pub async fn spawn_node(
                 }
             }
 
-            command.spawn().wrap_err(format!(
-                "failed to run runtime {}/{}",
-                runtime_config.node.dataflow_id, runtime_config.node.node_id
-            ))?
+            command
+                .stdin(Stdio::null())
+                .stdout(Stdio::piped())
```

Comment:
Just an idea: It's also possible to set a file directly as stdout/stderr target: https://doc.rust-lang.org/stable/std/process/struct.Stdio.html#impl-From%3CFile%3E-for-Stdio . I don't think that we can write stdout and stderr to the same file this way though...